Jenkinsfile

```diff
@@ -404,9 +404,10 @@ pipeline {
               agentLabel: 'chrome_112',
               runSteps: {
                 cambpmRunMaven('qa/',
-                  'clean install -Pwildfly,h2,webapps-integration',
+                  "clean install -Pwildfly,h2,webapps-integration -pl '!integration-tests-engine-jakarta'",
```

Comment:
ℹ️ This CI had to be updated to jdk17, which automatically enabled a profile that adds `integration-tests-engine-jakarta'`. For now we're excluding it manually here as it interfered with the Wildfly test setup, but I've created https://github.com/camunda/team-automation-platform/issues/282 to reconsider the logic behind this auto profile.